main.py

```diff
@@ -15,10 +15,128 @@
 import docker
 import teamcity
 import neo4j
-
+import argparse
 
 networks = ["the-bridge"]
 
+TestFlags = {
+    "TESTKIT_TESTS": True,
+    "UNIT_TESTS": True,
+    "INTEGRATION_TESTS": True,
+    "STUB_TESTS": True,
+    "STRESS_TESTS": True,
+    "TLS_TESTS": True
+}
+
+ConfigurationsToRun = []
+
+Configurations = [{"name": "4.2-cluster",
+                   "image": "neo4j:4.2-enterprise",
+                   "version": "4.2",
+                   "edition": "enterprise",
+                   "cluster": True,
+                   "suite": "",  # TODO: Define cluster suite
+                   "scheme": "neo4j"},
+                  {"name": "3.5-enterprise",
+                   "image": "neo4j:3.5-enterprise",
+                   "version": "3.5",
+                   "edition": "enterprise",
+                   "cluster": False,
+                   "suite": "3.5",
+                   "scheme": "bolt"},
+                  {"name": "4.0-community",
+                   "image": "neo4j:4.0",
+                   "version": "4.0",
+                   "edition": "community",
+                   "cluster": False,
+                   "suite": "4.0",
+                   "scheme": "neo4j"},
+                  {"name": "4.1-enterprise",
+                   "image": "neo4j:4.1-enterprise",
+                   "version": "4.1",
+                   "edition": "enterprise",
+                   "cluster": False,
+                   "suite": "4.1",
+                   "scheme": "neo4j"},
+                  {"name": "4.2-tc-enterprise",
```

Comment:
This one should only be added when teamcity is enabled. Otherwise it will fail strangely when running on host with default setup without having configured teamcity

Reply:
This is added to the configurations_to_run list only when in_teamcity.  This initial declaration is a 'master' list of all configs that are then selected from in the construct_configuration_list function.